key-uglifier/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "key-uglifier"
+version = "0.1.0"
+edition = "2021"
```

Comment:
2022?

Reply:
That's not the year, but rather the edition of Rust language: https://doc.rust-lang.org/edition-guide/rust-2021/index.html